packages/runtime/src/components/builtin/Form/components/Field/components/RadioButton/index.tsx

```diff
@@ -22,9 +22,10 @@ type ContainerBaseProps = Pick<Value, 'size'>
 type ContainerProps = ContainerBaseProps &
   Omit<ComponentPropsWithoutRef<'div'>, keyof ContainerBaseProps>
 
-function Container({ size }: ContainerProps) {
+function Container({ size, ...restOfProps }: ContainerProps) {
   return (
     <div
+      {...restOfProps}
```

Comment:
If we're gonna support all props and spread them instead of just adding `children`, we should also compose the class name prop